### ISSUE TITLE
Is it possible to define a custom loss function?

### ISSUE THREAD

**Issue by WangMengqi32C** [tensorflow/agents#619]
Dear,

I want to change the loss function a little bit,  just as this [https://arxiv.org/abs/2012.06 644].
Is this possible to this within the tf-agents?

Thank you in advance!

Best regards,
Mengqi
